src/Internal/Command/Invoker.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Internal\Command;
+
+use Rinq\Context;
+use Rinq\Ident\MessageId;
+use Rinq\Ident\PeerId;
+use Rinq\Payload;
```

Comment:
This payload keeps coming back :P  Payloads will just be a `mixed` type variable.